### ISSUE TITLE
Flake: TestDiagnose/bazel times out

### ISSUE THREAD

**Issue by briandealwis** [GoogleContainerTools/skaffold#5264]
[TestDiagnose/bazel](https://api.travis-ci.com/v3/job/473483182/log.txt) failure.  What's odd here is that from the output ("Ran in 2 minutes ...") the test seems to have succeeded, but goroutine 270 seems to indicate that it's still executing an external command.

Is it possible this test is running in parallel? (though wouldn't there be a log statement for the command?)

```
=== RUN   TestDiagnose/bazel
time="2021-01-20T17:35:47Z" level=info msg="[skaffold diagnose]"
time="2021-01-20T17:35:47Z" level=info msg="Ran in 2 minutes 0.245 second"
panic: test timed out after 20m0s
goroutine 240 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1618 +0xe5
created by time.goFunc
	/usr/local/go/src/time/sleep.go:167 +0x45

goroutine 1 [chan receive]:
testing.(*T).Run(0xc0008b9200, 0x214e68a, 0xc, 0x22ef668, 0x49b801)
	/usr/local/go/src/testing/testing.go:1169 +0x2da
testing.runTests.func1(0xc000109080)
	/usr/local/go/src/testing/testing.go:1439 +0x78
testing.tRunner(0xc000109080, 0xc0008ffde0)
	/usr/local/go/src/testing/testing.go:1123 +0xef
testing.runTests(0xc00031c1e0, 0x33628a0, 0x46, 0x46, 0xbffa385d196693d2, 0x11767f489d8, 0x337a580, 0x40f6b0)
	/usr/local/go/src/testing/testing.go:1437 +0x2fe
testing.(*M).Run(0xc00070c000, 0x0)
	/usr/local/go/src/testing/testing.go:1345 +0x1eb
main.main()
	_testmain.go:181 +0x138

goroutine 19 [chan receive]:
k8s.io/klog/v2.(*loggingT).flushDaemon(0x337ad60)
	/root/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:1131 +0x8b
created by k8s.io/klog/v2.init.0
	/root/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:416 +0xd8
goroutine 6 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x337abc0)
	/root/go/pkg/mod/github.com/golang/glog@v0.0.0-20160126235308-23def4e6c14b/glog.go:882 +0x8b
created by github.com/golang/glog.init.0
	/root/go/pkg/mod/github.com/golang/glog@v0.0.0-20160126235308-23def4e6c14b/glog.go:410 +0x274

goroutine 7 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc00070c700)
	/root/go/pkg/mod/go.opencensus.io@v0.22.5/stats/view/worker.go:276 +0x105
created by go.opencensus.io/stats/view.init.0
	/root/go/pkg/mod/go.opencensus.io@v0.22.5/stats/view/worker.go:34 +0x68

goroutine 213 [IO wait]:
internal/poll.runtime_pollWait(0x7f8e3ce36c30, 0x72, 0x24a18a0)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000930918, 0x72, 0x24a1800, 0x32b3ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc000930900, 0xc000a80000, 0x6986, 0x6986, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
net.(*netFD).Read(0xc000930900, 0xc000a80000, 0x6986, 0x6986, 0x203000, 0x7731bb, 0xc0003b64e0)
	/usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc000010058, 0xc000a80000, 0x6986, 0x6986, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:182 +0x8e
crypto/tls.(*atLeastReader).Read(0xc0002de140, 0xc000a80000, 0x6986, 0x6986, 0x267, 0x6946, 0xc000079710)
	/usr/local/go/src/crypto/tls/conn.go:779 +0x62
bytes.(*Buffer).ReadFrom(0xc0003b6600, 0x2499260, 0xc0002de140, 0x40d3c5, 0x1e495a0, 0x20b94e0)
	/usr/local/go/src/bytes/buffer.go:204 +0xb1
crypto/tls.(*Conn).readFromUntil(0xc0003b6380, 0x249ef20, 0xc000010058, 0x5, 0xc000010058, 0x256)
	/usr/local/go/src/crypto/tls/conn.go:801 +0xf3
crypto/tls.(*Conn).readRecordOrCCS(0xc0003b6380, 0x0, 0x0, 0xc000079d18)
	/usr/local/go/src/crypto/tls/conn.go:608 +0x115
crypto/tls.(*Conn).readRecord(...)
	/usr/local/go/src/crypto/tls/conn.go:576
crypto/tls.(*Conn).Read(0xc0003b6380, 0xc0004df000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1252 +0x15f
bufio.(*Reader).Read(0xc00009bbc0, 0xc000116498, 0x9, 0x9, 0xc000079d18, 0x22f3e00, 0x97096b)
	/usr/local/go/src/bufio/bufio.go:227 +0x222
io.ReadAtLeast(0x2499000, 0xc00009bbc0, 0xc000116498, 0x9, 0x9, 0x9, 0xc000114030, 0x0, 0x24994c0)
	/usr/local/go/src/io/io.go:314 +0x87
io.ReadFull(...)
	/usr/local/go/src/io/io.go:333
golang.org/x/net/http2.readFrameHeader(0xc000116498, 0x9, 0x9, 0x2499000, 0xc00009bbc0, 0x0, 0x0, 0xc0004360c0, 0x0)
	/root/go/pkg/mod/golang.org/x/net@v0.0.0-20201202161906-c7110b5ffcbb/http2/frame.go:237 +0x89
golang.org/x/net/http2.(*Framer).ReadFrame(0xc000116460, 0xc0004360c0, 0x0, 0x0, 0x0)
	/root/go/pkg/mod/golang.org/x/net@v0.0.0-20201202161906-c7110b5ffcbb/http2/frame.go:492 +0xa5
golang.org/x/net/http2.(*clientConnReadLoop).run(0xc000079fa8, 0x0, 0x0)
	/root/go/pkg/mod/golang.org/x/net@v0.0.0-20201202161906-c7110b5ffcbb/http2/transport.go:1819 +0xd8
golang.org/x/net/http2.(*ClientConn).readLoop(0xc000582f00)
	/root/go/pkg/mod/golang.org/x/net@v0.0.0-20201202161906-c7110b5ffcbb/http2/transport.go:1741 +0x6f
created by golang.org/x/net/http2.(*Transport).newClientConn
	/root/go/pkg/mod/golang.org/x/net@v0.0.0-20201202161906-c7110b5ffcbb/http2/transport.go:705 +0x6c5

goroutine 9 [chan receive, 20 minutes]:
github.com/GoogleContainerTools/skaffold/pkg/skaffold/event.newHandler.func1(0xc0000e2580)
	/skaffold/pkg/skaffold/event/event.go:59 +0x45
created by github.com/GoogleContainerTools/skaffold/pkg/skaffold/event.newHandler
	/skaffold/pkg/skaffold/event/event.go:57 +0x90

goroutine 10 [chan receive, 20 minutes]:
github.com/rjeczalik/notify.(*nonrecursiveTree).dispatch(0xc00009b680, 0xc00009b500)
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:36 +0xb6
created by github.com/rjeczalik/notify.newNonrecursiveTree
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:29 +0xe5

goroutine 11 [chan receive, 20 minutes]:
github.com/rjeczalik/notify.(*nonrecursiveTree).internal(0xc00009b680, 0xc00009b5c0)
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:81 +0x58
created by github.com/rjeczalik/notify.newNonrecursiveTree
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:30 +0x111

goroutine 269 [chan receive]:
testing.(*T).Run(0xc0008b9380, 0xc0009cc7d9, 0x5, 0xc000a5e8d0, 0x20)
	/usr/local/go/src/testing/testing.go:1169 +0x2da
github.com/GoogleContainerTools/skaffold/integration.TestDiagnose(0xc0008b9200)
	/skaffold/integration/diagnose_test.go:38 +0x172
testing.tRunner(0xc0008b9200, 0x22ef668)
	/usr/local/go/src/testing/testing.go:1123 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1168 +0x2b3

goroutine 221 [chan send, 4 minutes]:
k8s.io/apimachinery/pkg/watch.(*StreamWatcher).receive(0xc000b3f240)
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:127 +0x12d
created by k8s.io/apimachinery/pkg/watch.NewStreamWatcher
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:71 +0xbe

goroutine 271 [IO wait]:
internal/poll.runtime_pollWait(0x7f8e3ce36978, 0x72, 0x24a18a0)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000334d98, 0x72, 0x24a1801, 0x32b3ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc000334d80, 0xc00066929b, 0x565, 0x565, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
os.(*File).read(...)
	/usr/local/go/src/os/file_posix.go:31
os.(*File).Read(0xc00042cb08, 0xc00066929b, 0x565, 0x565, 0x9b, 0x0, 0x0)
	/usr/local/go/src/os/file.go:116 +0x71
bytes.(*Buffer).ReadFrom(0xc000660660, 0x249f320, 0xc00042cb08, 0x7f8e14305a58, 0xc000660660, 0x1)
	/usr/local/go/src/bytes/buffer.go:204 +0xb1
io.copyBuffer(0x2499060, 0xc000660660, 0x249f320, 0xc00042cb08, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc0000f57c0)
	/usr/local/go/src/io/io.go:395 +0x2ff
io.Copy(...)
	/usr/local/go/src/io/io.go:368
os/exec.(*Cmd).writerDescriptor.func1(0xc0008b9380, 0x22f3fe8)
	/usr/local/go/src/os/exec/exec.go:311 +0x65
os/exec.(*Cmd).Start.func1(0xc000b60160, 0xc000475360)
	/usr/local/go/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:440 +0x629

goroutine 272 [IO wait]:
internal/poll.runtime_pollWait(0x7f8e3ce36b48, 0x72, 0x24a18a0)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000334f18, 0x72, 0x24a1801, 0x32b3ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc000334f00, 0xc000682000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
os.(*File).read(...)
	/usr/local/go/src/os/file_posix.go:31
os.(*File).Read(0xc00042cb20, 0xc000682000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/os/file.go:116 +0x71
io.copyBuffer(0x249f440, 0xc00064ecd0, 0x249f320, 0xc00042cb20, 0xc000682000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:409 +0x12c
io.Copy(...)
	/usr/local/go/src/io/io.go:368
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:311 +0x65
os/exec.(*Cmd).Start.func1(0xc000b60160, 0xc000475420)
	/usr/local/go/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:440 +0x629

goroutine 273 [select]:
os/exec.(*Cmd).Start.func2(0xc000b60160)
	/usr/local/go/src/os/exec/exec.go:449 +0xd5
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:448 +0x6c5

goroutine 270 [syscall]:
syscall.Syscall6(0xf7, 0x1, 0x12da, 0xc0006bbad0, 0x1000004, 0x0, 0x0, 0x4b1201, 0xc000334fc0, 0xc0006bbb10)
	/usr/local/go/src/syscall/asm_linux_amd64.s:41 +0x5
os.(*Process).blockUntilWaitable(0xc000510690, 0x203000, 0x0, 0x0)
	/usr/local/go/src/os/wait_waitid.go:32 +0xa6
os.(*Process).wait(0xc000510690, 0x22f3880, 0x22f3888, 0x22f3878)
	/usr/local/go/src/os/exec_unix.go:22 +0x39
os.(*Process).Wait(...)
	/usr/local/go/src/os/exec.go:125
os/exec.(*Cmd).Wait(0xc000b60160, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:507 +0x65
os/exec.(*Cmd).Run(0xc000b60160, 0xc00064ecd0, 0x2fb71770005ac3a5)
	/usr/local/go/src/os/exec/exec.go:341 +0x5c
os/exec.(*Cmd).Output(0xc000b60160, 0x11742aa4353, 0x337a580, 0x1, 0x1, 0x106)
	/usr/local/go/src/os/exec/exec.go:546 +0x92
github.com/GoogleContainerTools/skaffold/integration/skaffold.(*RunBuilder).RunOrFailOutput(0xc0006bbec0, 0xc0008b9380, 0x0, 0x0, 0x0)
	/skaffold/integration/skaffold/helper.go:275 +0x1a9
github.com/GoogleContainerTools/skaffold/integration/skaffold.(*RunBuilder).RunOrFail(...)
	/skaffold/integration/skaffold/helper.go:229
github.com/GoogleContainerTools/skaffold/integration.TestDiagnose.func1(0xc0008b9380)
	/skaffold/integration/diagnose_test.go:45 +0x1e5
testing.tRunner(0xc0008b9380, 0xc000a5e8d0)
	/usr/local/go/src/testing/testing.go:1123 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1168 +0x2b3

goroutine 71 [IO wait, 19 minutes]:
internal/poll.runtime_pollWait(0x7f8e3ce36d18, 0x72, 0x24a18a0)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0007d6718, 0x72, 0x24a1800, 0x32b3ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0007d6700, 0xc000800000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
net.(*netFD).Read(0xc0007d6700, 0xc000800000, 0x1000, 0x1000, 0x43c5dc, 0xc00007cb58, 0x469680)
	/usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc000122e88, 0xc000800000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:182 +0x8e
net/http.(*persistConn).Read(0xc0007f06c0, 0xc000800000, 0x1000, 0x1000, 0xc0007f4240, 0xc00007cc58, 0x407555)
	/usr/local/go/src/net/http/transport.go:1887 +0x77
bufio.(*Reader).fill(0xc00075f320)
	/usr/local/go/src/bufio/bufio.go:101 +0x105
bufio.(*Reader).Peek(0xc00075f320, 0x1, 0x0, 0x0, 0x1, 0x0, 0xc000745320)
	/usr/local/go/src/bufio/bufio.go:139 +0x4f
net/http.(*persistConn).readLoop(0xc0007f06c0)
	/usr/local/go/src/net/http/transport.go:2040 +0x1a8
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1708 +0xcb7

goroutine 72 [select, 19 minutes]:
net/http.(*persistConn).writeLoop(0xc0007f06c0)
	/usr/local/go/src/net/http/transport.go:2340 +0x11c
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1709 +0xcdc
FAIL	github.com/GoogleContainerTools/skaffold/integration	1200.071s
```

**Comment by briandealwis** [comment]
In `TestDiagnose/jib-multimodule` this time [log](https://api.travis-ci.com/v3/job/473985115/log.txt).  There's no `Ran in` this time, so at least the hang looks "normal".
```
=== RUN   TestDiagnose/jib-multimodule
time="2021-01-21T20:16:41Z" level=info msg="Running [skaffold diagnose] in examples/jib-multimodule"
panic: test timed out after 20m0s
goroutine 382 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1618 +0xe5
created by time.goFunc
	/usr/local/go/src/time/sleep.go:167 +0x45
goroutine 1 [chan receive]:
testing.(*T).Run(0xc000c7f080, 0x215077a, 0xc, 0x22f1a20, 0x49b801)
	/usr/local/go/src/testing/testing.go:1169 +0x2da
testing.runTests.func1(0xc00010a180)
	/usr/local/go/src/testing/testing.go:1439 +0x78
testing.tRunner(0xc00010a180, 0xc0008ffde0)
	/usr/local/go/src/testing/testing.go:1123 +0xef
testing.runTests(0xc0002de260, 0x3365960, 0x46, 0x46, 0xbffa962ce3d5186d, 0x11768700730, 0x337d760, 0x40f6b0)
	/usr/local/go/src/testing/testing.go:1437 +0x2fe
testing.(*M).Run(0xc0004d2080, 0x0)
	/usr/local/go/src/testing/testing.go:1345 +0x1eb
main.main()
	_testmain.go:181 +0x138
goroutine 6 [chan receive]:
k8s.io/klog/v2.(*loggingT).flushDaemon(0x337df40)
	/root/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:1131 +0x8b
created by k8s.io/klog/v2.init.0
	/root/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:416 +0xd8
goroutine 19 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x337dda0)
	/root/go/pkg/mod/github.com/golang/glog@v0.0.0-20160126235308-23def4e6c14b/glog.go:882 +0x8b
created by github.com/golang/glog.init.0
	/root/go/pkg/mod/github.com/golang/glog@v0.0.0-20160126235308-23def4e6c14b/glog.go:410 +0x274
goroutine 9 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc0004d2780)
	/root/go/pkg/mod/go.opencensus.io@v0.22.5/stats/view/worker.go:276 +0x105
created by go.opencensus.io/stats/view.init.0
	/root/go/pkg/mod/go.opencensus.io@v0.22.5/stats/view/worker.go:34 +0x68
goroutine 320 [chan receive]:
testing.(*T).Run(0xc000ad0300, 0xc000a83449, 0xf, 0xc000bb49a0, 0x1)
	/usr/local/go/src/testing/testing.go:1169 +0x2da
github.com/GoogleContainerTools/skaffold/integration.TestDiagnose(0xc000c7f080)
	/skaffold/integration/diagnose_test.go:38 +0x172
testing.tRunner(0xc000c7f080, 0x22f1a20)
	/usr/local/go/src/testing/testing.go:1123 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1168 +0x2b3
goroutine 11 [chan receive, 20 minutes]:
github.com/GoogleContainerTools/skaffold/pkg/skaffold/event.newHandler.func1(0xc0005e6420)
	/skaffold/pkg/skaffold/event/event.go:59 +0x45
created by github.com/GoogleContainerTools/skaffold/pkg/skaffold/event.newHandler
	/skaffold/pkg/skaffold/event/event.go:57 +0x90
goroutine 12 [chan receive, 20 minutes]:
github.com/rjeczalik/notify.(*nonrecursiveTree).dispatch(0xc0003c98c0, 0xc0003c9800)
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:36 +0xb6
created by github.com/rjeczalik/notify.newNonrecursiveTree
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:29 +0xe5
goroutine 13 [chan receive, 20 minutes]:
github.com/rjeczalik/notify.(*nonrecursiveTree).internal(0xc0003c98c0, 0xc0003c9860)
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:81 +0x58
created by github.com/rjeczalik/notify.newNonrecursiveTree
	/root/go/pkg/mod/github.com/rjeczalik/notify@v0.9.2/tree_nonrecursive.go:30 +0x111
goroutine 262 [chan send, 3 minutes]:
k8s.io/apimachinery/pkg/watch.(*StreamWatcher).receive(0xc0006e8ec0)
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:127 +0x12d
created by k8s.io/apimachinery/pkg/watch.NewStreamWatcher
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:71 +0xbe
goroutine 73 [select, 19 minutes]:
net/http.(*persistConn).writeLoop(0xc0009e0240)
	/usr/local/go/src/net/http/transport.go:2340 +0x11c
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1709 +0xcdc
goroutine 378 [syscall]:
syscall.Syscall6(0xf7, 0x1, 0x5970, 0xc000e45aa0, 0x1000004, 0x0, 0x0, 0x4b1201, 0xc000bd8780, 0xc000e45ae0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:41 +0x5
os.(*Process).blockUntilWaitable(0xc000014de0, 0x203000, 0x0, 0x0)
	/usr/local/go/src/os/wait_waitid.go:32 +0xa6
os.(*Process).wait(0xc000014de0, 0x22f5c40, 0x22f5c48, 0x22f5c38)
	/usr/local/go/src/os/exec_unix.go:22 +0x39
os.(*Process).Wait(...)
	/usr/local/go/src/os/exec.go:125
os/exec.(*Cmd).Wait(0xc0008f0420, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:507 +0x65
os/exec.(*Cmd).Run(0xc0008f0420, 0xc000bc0a50, 0x1f0426ef0210fa20)
	/usr/local/go/src/os/exec/exec.go:341 +0x5c
os/exec.(*Cmd).Output(0xc0008f0420, 0x1150f9331a5, 0x337d760, 0x10, 0xc000e45d90, 0x2)
	/usr/local/go/src/os/exec/exec.go:546 +0x92
github.com/GoogleContainerTools/skaffold/integration/skaffold.(*RunBuilder).RunOrFailOutput(0xc000e45ec0, 0xc000ad0300, 0x0, 0x0, 0x0)
	/skaffold/integration/skaffold/helper.go:275 +0x216
github.com/GoogleContainerTools/skaffold/integration/skaffold.(*RunBuilder).RunOrFail(...)
	/skaffold/integration/skaffold/helper.go:229
github.com/GoogleContainerTools/skaffold/integration.TestDiagnose.func1(0xc000ad0300)
	/skaffold/integration/diagnose_test.go:45 +0x1e5
testing.tRunner(0xc000ad0300, 0xc000bb49a0)
	/usr/local/go/src/testing/testing.go:1123 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1168 +0x2b3
goroutine 72 [IO wait, 19 minutes]:
internal/poll.runtime_pollWait(0x7f5c0c5bc230, 0x72, 0x24a3f20)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0009d6e18, 0x72, 0x24a3f00, 0x32b6ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0009d6e00, 0xc0007b7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
net.(*netFD).Read(0xc0009d6e00, 0xc0007b7000, 0x1000, 0x1000, 0x43c5dc, 0xc00007eb58, 0x469680)
	/usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc00042dec8, 0xc0007b7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:182 +0x8e
net/http.(*persistConn).Read(0xc0009e0240, 0xc0007b7000, 0x1000, 0x1000, 0xc0004e79e0, 0xc00007ec58, 0x407555)
	/usr/local/go/src/net/http/transport.go:1887 +0x77
bufio.(*Reader).fill(0xc0009cb020)
	/usr/local/go/src/bufio/bufio.go:101 +0x105
bufio.(*Reader).Peek(0xc0009cb020, 0x1, 0x0, 0x0, 0x1, 0x0, 0xc00009da40)
	/usr/local/go/src/bufio/bufio.go:139 +0x4f
net/http.(*persistConn).readLoop(0xc0009e0240)
	/usr/local/go/src/net/http/transport.go:2040 +0x1a8
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1708 +0xcb7
goroutine 381 [select]:
os/exec.(*Cmd).Start.func2(0xc0008f0420)
	/usr/local/go/src/os/exec/exec.go:449 +0xd5
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:448 +0x6c5
goroutine 151 [chan send, 10 minutes]:
k8s.io/apimachinery/pkg/watch.(*StreamWatcher).receive(0xc0007207c0)
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:127 +0x12d
created by k8s.io/apimachinery/pkg/watch.NewStreamWatcher
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:71 +0xbe
goroutine 288 [select]:
net/http.(*persistConn).writeLoop(0xc0003b2a20)
	/usr/local/go/src/net/http/transport.go:2340 +0x11c
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1709 +0xcdc
goroutine 379 [IO wait]:
internal/poll.runtime_pollWait(0x7f5c0c5bbf78, 0x72, 0x24a3f20)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000bd8618, 0x72, 0x24a3f01, 0x32b6ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc000bd8600, 0xc000be8099, 0x567, 0x567, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
os.(*File).read(...)
	/usr/local/go/src/os/file_posix.go:31
os.(*File).Read(0xc00030d688, 0xc000be8099, 0x567, 0x567, 0x99, 0x0, 0x0)
	/usr/local/go/src/os/file.go:116 +0x71
bytes.(*Buffer).ReadFrom(0xc0006a8990, 0x24a19a0, 0xc00030d688, 0x7f5c0c56dae0, 0xc0006a8990, 0xc000979f01)
	/usr/local/go/src/bytes/buffer.go:204 +0xb1
io.copyBuffer(0x249b6e0, 0xc0006a8990, 0x24a19a0, 0xc00030d688, 0x0, 0x0, 0x0, 0x407555, 0xc000ba4c00, 0xc000979fb0)
	/usr/local/go/src/io/io.go:395 +0x2ff
io.Copy(...)
	/usr/local/go/src/io/io.go:368
os/exec.(*Cmd).writerDescriptor.func1(0xc000ba4c00, 0xc000979fb0)
	/usr/local/go/src/os/exec/exec.go:311 +0x65
os/exec.(*Cmd).Start.func1(0xc0008f0420, 0xc0006d0b40)
	/usr/local/go/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:440 +0x629
goroutine 264 [chan send, 3 minutes]:
k8s.io/apimachinery/pkg/watch.(*StreamWatcher).receive(0xc0006e9380)
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:127 +0x12d
created by k8s.io/apimachinery/pkg/watch.NewStreamWatcher
	/root/go/pkg/mod/k8s.io/apimachinery@v0.19.4/pkg/watch/streamwatcher.go:71 +0xbe
goroutine 380 [IO wait]:
internal/poll.runtime_pollWait(0x7f5c0c5bc318, 0x72, 0x24a3f20)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000bd86d8, 0x72, 0x24a3f01, 0x32b6ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc000bd86c0, 0xc000c1a000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
os.(*File).read(...)
	/usr/local/go/src/os/file_posix.go:31
os.(*File).Read(0xc00030d6a0, 0xc000c1a000, 0x8000, 0x8000, 0x0, 0x0, 0xc0000f6e08)
	/usr/local/go/src/os/file.go:116 +0x71
io.copyBuffer(0x24a1ac0, 0xc000bc0a50, 0x24a19a0, 0xc00030d6a0, 0xc000c1a000, 0x8000, 0x8000, 0x1, 0xc000a31e00, 0x1000000010000)
	/usr/local/go/src/io/io.go:409 +0x12c
io.Copy(...)
	/usr/local/go/src/io/io.go:368
os/exec.(*Cmd).writerDescriptor.func1(0x22f5ee0, 0x0)
	/usr/local/go/src/os/exec/exec.go:311 +0x65
os/exec.(*Cmd).Start.func1(0xc0008f0420, 0xc0006d0b80)
	/usr/local/go/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:440 +0x629
goroutine 287 [IO wait]:
internal/poll.runtime_pollWait(0x7f5c0c5bbcc0, 0x72, 0x24a3f20)
	/usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0000ea798, 0x72, 0x24a3f00, 0x32b6ae8, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0000ea780, 0xc0004d8800, 0x1716, 0x1716, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:159 +0x1a5
net.(*netFD).Read(0xc0000ea780, 0xc0004d8800, 0x1716, 0x1716, 0x203000, 0x7731bb, 0xc0002b64e0)
	/usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc000010e98, 0xc0004d8800, 0x1716, 0x1716, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:182 +0x8e
crypto/tls.(*atLeastReader).Read(0xc00031cae0, 0xc0004d8800, 0x1716, 0x1716, 0x2ed, 0x1711, 0xc000b6a668)
	/usr/local/go/src/crypto/tls/conn.go:779 +0x62
bytes.(*Buffer).ReadFrom(0xc0002b6600, 0x249b8e0, 0xc00031cae0, 0x40d3c5, 0x1e4b640, 0x20bb5a0)
	/usr/local/go/src/bytes/buffer.go:204 +0xb1
crypto/tls.(*Conn).readFromUntil(0xc0002b6380, 0x24a15a0, 0xc000010e98, 0x5, 0xc000010e98, 0x2dc)
	/usr/local/go/src/crypto/tls/conn.go:801 +0xf3
crypto/tls.(*Conn).readRecordOrCCS(0xc0002b6380, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:608 +0x115
crypto/tls.(*Conn).readRecord(...)
	/usr/local/go/src/crypto/tls/conn.go:576
crypto/tls.(*Conn).Read(0xc0002b6380, 0xc000c7c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1252 +0x15f
net/http.(*persistConn).Read(0xc0003b2a20, 0xc000c7c000, 0x1000, 0x1000, 0xc00009dce0, 0xc000b6ac58, 0x407555)
	/usr/local/go/src/net/http/transport.go:1887 +0x77
bufio.(*Reader).fill(0xc0009cacc0)
	/usr/local/go/src/bufio/bufio.go:101 +0x105
bufio.(*Reader).Peek(0xc0009cacc0, 0x1, 0x0, 0x0, 0x1, 0x0, 0xc0008b8840)
	/usr/local/go/src/bufio/bufio.go:139 +0x4f
net/http.(*persistConn).readLoop(0xc0003b2a20)
	/usr/local/go/src/net/http/transport.go:2040 +0x1a8
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:1708 +0xcb7
FAIL	github.com/GoogleContainerTools/skaffold/integration	1200.063s
```

**Comment by briandealwis** [comment]
The overall suite of tests are taking too long (https://github.com/GoogleContainerTools/skaffold/issues/5261#issuecomment-765672504).

Duplicate of #5261
